### PR TITLE
Add changelog of v2.24.0

### PR DESCRIPTION
**Pull request by ZwwWayne**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Bump version to v2.24.0

## Modification

docs/en/changelog.md

## BC-breaking (Optional)

N/A

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/changelog.md [hunk 1]
```diff
@@ -1,5 +1,127 @@
...
+  in the config to use `ClassAwareSampler`. Examples can be found in [the configs of OpenImages Dataset](https://github.com/open-mmlab/mmdetection/tree/master/configs/openimages/faster_rcnn_r50_fpn_32x2_cas_1x_openimages.py)  (#7436)
+
+- Support automatically scaling LR according to GPU number and samples per GPU. In each config, there is a corresponding config of auto-scaling LR as below,
+
+   ```python
+   auto_scale_lr = dict(enable=True, base_batch_size=N)
+   ```
+
+  where `N` is the batch size used for the current learning rate in the config (also equals to `samples_per_gpu` * gpu number to train this config).
+  By default, we set `enable=False` so that the original usages will not be affected. Users can set `enable=True` in each config to enable this feature and should check the correctness of `base_batch_size` in customized configs. (#7482)
```
Comment (jbwang1997) [inline docs/en/changelog.md:29]: Users can set `enable=True` in each config or add `--auto-scale-lr` after the command line to enable this feature and should check the correctness of `base_batch_size` in customized configs.

File: docs/en/changelog.md [hunk 2]
```diff
@@ -1,5 +1,127 @@
...
+    ```
+
+- Support to run on PyTorch with MLU chip (#7578)
+- Support re-spliting data batch with tag (#7641)
+- Support the `DiceCost` used by [K-Net](https://arxiv.org/abs/2106.14855) in `MaskHungarianAssigner` (#7716)
+- Support splitting COCO data for Semi-supervised object detection. (#7431)
+- Support Pathlib for Config.fromfile (#7685)
+- Support to use file client in OpenImages dataset (#7433)
+- Add a probability parameter to Mosaic transformation (#7371)
+- Support specifying interpolation mode of resize in transforms.py (#7585)
```
Comment (jbwang1997) [inline docs/en/changelog.md:93]: Support specifying interpolation mode of `Resize` in transforms.py

**Comment by jbwang1997** [comment]
Lint is not passed. We need to fix the lint error.

**Review by hhaAndroid — COMMENTED** [review]
LGTM

**Review by jbwang1997 — APPROVED** [review]
LGTM

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**